wagtail/images/templates/wagtailimages/chooser/confirm_duplicate_upload.html

```diff
@@ -0,0 +1,37 @@
+{% load i18n wagtailadmin_tags wagtailimages_tags %}
+
+{% block extra_css %}
+    <link rel="stylesheet" href="{% versioned_static 'wagtailimages/css/chooser-duplicate-upload.css' %}" type="text/css" />
+{% endblock %}
+
+<section id="upload" class="duplicate-upload active nice-padding">
+    <p class="help-block help-warning">
+        {% icon name='warning' %}
+        {% trans "Upload successful. However, your new image seems to be a duplicate of an existing image. You may delete it if it wasn't required." %}
+    </p>
+
+    <figure class="duplicate-upload__figure col6">
+        <figcaption>Existing</figcaption>
+        {% image existing_image max-800x600 as img %}
+        <img src="{{ img.url }}" width="{{ img.width }}" height="{{ img.height }}" alt="{{ img.alt }}" class="show-transparency"/>
```

Comment:
Not sure why we're generating the tags by hand here rather than using `{% image %}` directly?